packages/trader/build/loaders-config.js

```diff
@@ -1,6 +1,7 @@
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const path = require('path');
-const { IS_RELEASE } = require('./constants');
+
+const IS_RELEASE = process.env.NODE_ENV === 'production' || process.env.NODE_ENV === 'staging';
```

Comment:
@jim-deriv Not related to this PR, But this can be a method in `@deriv/utils` maybe 🤔 

```ts
const getEnviroment = () => {
  const is_production = process.env.NODE_ENV === 'production';
  const is_staging = process.env.NODE_ENV === 'staging';
  const is_release = is_production || is_staging;

  return {
    is_production,
    is_staging,
    is_release,
  };
}
```

```ts
import { getEnviroment } from '@deriv/utils';

const { is_production, is_staging, is_release } = getEnviroment();
```

Reply:
@farzin-deriv, sure will move it in another PR